cve_bin_tool/checkers/__init__.py

```diff
@@ -12,6 +12,7 @@
     "Checker",
     "VendorProductPair",
     "accountsservice",
+    "apache",
```

Comment:
```suggestion
    "apache_http_server",
```
Apache alone is not good as a checker name since there could be other Apache products.